### PR TITLE
Homework8: Vladislav Savchuk

### PR DESCRIPTION
**Pull request by VladislavSavchuk**
# Decisions of tasks homework #8

link [file:///D:/IT%20Academy%20Automation%20QA/%D0%94%D0%97/HW%238.html](url)

- Sequence
- Opposite number
- Validate

### CODE REVIEW

**Comment by VladislavSavchuk** [comment]
Hello! @Stml89 made review homework #8, please

File: homework/homework8/number_opposite.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+    # Добавляем это значение к first_number
+    opposite_number = half_circle + first_number
+
+    # Находим остаток от деления полученного значения на 'n'
+    result = opposite_number % n
+    print(result)
+
+
+main(10, 2)
```
Comment (Stml89) [inline homework/homework8/number_opposite.py]: it would be better if you add assert

**Review by Stml89 — COMMENTED** [review]
fix comment and you can merge PR

**Comment by VladislavSavchuk** [comment]
fixed the code according to the comment

**Pull request merged**